src/Data/Dependent/Sum/Orphans.hs

```diff
@@ -3,6 +3,7 @@
 {-# LANGUAGE TypeOperators #-}
 {-# LANGUAGE UndecidableInstances #-}
 {-# LANGUAGE TypeApplications #-}
+{-# LANGUAGE PatternSynonyms #-}
```

Comment:
Please sort.

Reply:
That extension doesn‘t fix anything, so I revert it …